.gitlab-ci.yml

```diff
@@ -211,10 +242,8 @@ ci-equations:
 ci-fcsl-pcm:
   <<: *ci-template
 
-# ci-fiat-crypto:
-#   <<: *ci-template
-#   # out of memory error
-#   allow_failure: true
+ci-fiat-crypto:
+  <<: *ci-template
```

Comment:
Why are you resurrecting fiat-crypto?

Reply:
Why not? As far as I can tell it should work fine , workers have more than enough ram.

The idea of this PR was actually to allow to remove all Travis jobs but for the linter and the Mac one, so this is the reason I add it back.

Reply:
I can document better if you want.

Reply:
In fact I should add the lint job too so we are complete?

Reply:
Maybe the workers got upgraded?

Reply:
The specs are pretty beefy these days, so if you think that's OK I'll add the linter job to this PR so we have a complete Travis replacement just in case [except for OSX, but that's a different story].

Reply:
The main reason the linter isn't on gitlab is that it uses the TRAVIS_COMMIT_RANGE variable. There's no equivalent on gitlab, especially considering we run as a normal branch job.
Do you have a plan to deal with that?

Reply:
Nope, in fact I ported everything but the linter due to it looking complicated. So let's the linter remain there for now, if Travis becomes a linter-only job for PR that's fine [as we can interpret the red light as a certain linter failure then]

Reply:
In `oexp` I used, `git rev-list --ancestry-path %s..%s`, to get a commit range, I dunno if that could work here.